examples/core/core_input_gamepad.c

```diff
@@ -163,7 +163,7 @@ int main(void)
 
                 DrawText(FormatText("DETECTED AXIS [%i]:", GetGamepadAxisCount(GAMEPAD_PLAYER1)), 10, 50, 10, MAROON);
 
-                for (int i = 0; i < GetGamepadAxisCount(GAMEPAD_PLAYER1); i++)
+                for (int i = 1; i <= GetGamepadAxisCount(GAMEPAD_PLAYER1); i++)
```

Comment:
This looks really counter-intuitive... it should return the number of axis for indexing starting at 0...

Reply:
0 corresponds to GAMEPAD_AXIS_UNKNOWN.
I can leave it like this and do i + 1 inside of the loop. But for me it's not intuitive too...

Reply:
@mmalecot Checking [the documentation](https://www.glfw.org/docs/3.3/group__gamepad__axes.html), gamepad axis indices start at 0 and GLFW_GAMEPAD_AXIS_LAST it's the count.

Reply:
@raysan5 Yes that's right on the GLFW side. But on [raylib side](https://github.com/raysan5/raylib/blob/master/src/raylib.h#L667), 0 is used for error checking and axes begin at 1.

If you try the core_input_gamepad example without this PR, you will see that Axis 0 has no reality.

Reply:
@mmalecot I see... then issue is probably there, not in the axis count... counting one extra unexistent axis to solve an indexing shift shouldn't be the solution...

Reply:
@raysan5 OK, see below for details of the last modification.